bin/primer_designer_region.py

```diff
@@ -1319,12 +1327,14 @@ def pretty_pdf_fusion_mappings(
                 target_sequence = coord_dict[0]['SEQ']
                 tagged_string = coord_dict[0]['TSEQ']
 
-                if coord_dict[0]['SIDE'] == "<" and coord_dict[0]['STRAND'] == "-1":
+                if coord_dict[0]['SIDE'] == "<" and \
```

Comment:
I think this is still against pep8 as the following line is wrongly indented, this is more correct I think:

```
if (coord_dict[0]['SIDE'] == "<" and
    coord_dict[0]['STRAND'] == "-1"
    ):
    base1 = int(coord_dict[0]['POS']) + FLANK
```

https://peps.python.org/pep-0008/#indentation

Reply:
changed all to use bracket rather than \